CONTRIBUTING.md

```diff
@@ -3,3 +3,9 @@
 To contribute to Zinc, you or your employer must sign Lightbend Contributor License Agreement <https://www.lightbend.com/contribute/cla>.
 
 If you add a new file, run `createHeaders` task from sbt shell to put the copyright notice.
+
+### To build with other module source
```

Comment:
What does this mean for Zinc devs? Why would we want this?
  
I know there are a lot of moving parts, but documenting that would be nice!